Dockerfile

```diff
@@ -1,44 +1,29 @@
-FROM quay.io/centos/centos:stream8 as stream8
+FROM quay.io/almalinux/almalinux:8 AS almalinux8
 
 ARG TARGETPLATFORM
 
+# Update the public key
+RUN rpm --import https://repo.almalinux.org/almalinux/RPM-GPG-KEY-AlmaLinux-8
 
 # Install base packages that are used across both the application and Vespa
 RUN dnf install -y epel-release dnf-utils ca-certificates curl gnupg && \
     dnf config-manager --set-enabled powertools
 
-
 # Install application specific packages
-RUN dnf groupinstall "Development Tools" -y && \
```

Comment:
only gcc is used in installation of python packages. This also introduce java 1.8 dep which messes with the openjdk-17 installation.